ocaml/xapi/xapi_vm.ml

```diff
@@ -221,6 +221,16 @@ let unpause ~__context ~vm =
 let set_xenstore_data ~__context ~self ~value =
   Xapi_xenops.set_xenstore_data ~__context ~self value
 
+(* CP-18860: check memory limits if using nested_virt on start *)
+let assert_memory_constraints ~__context ~vm platformdata =
```

Comment:
One of the existing tests in `assert_can_boot_here` is `validate_memory` which looks only at the VM metadata, not the host... but I suppose the `can_boot_here` _should_ be all about the `here`, so yes, I agree with Rob.